tests/unit/plugins/module_utils/test_xml_utils.py

```diff
@@ -440,10 +455,33 @@ def test_elements_equal_with_children():
     e1c2 = Element("child_2")
     e1c2.text = "some_text_as_well"
     e1.extend([e1c1, e1c2])
+
     e2 = Element("test")
     e2c1 = Element("child_1")
     e2c1.text = "some_text     \n"
     e2c2 = Element("child_2")
-    e2c2.text = "\n   some_text     \n"
+    e2c2.text = "\n   some_text_as_well     \n"
     e2.extend([e2c1, e2c2])
+
     assert xml_utils.elements_equal(e1, e2)
+
+
+def test_elements_not_equal_with_children():
+    """
+    Tests elements_equal function checks recursively elements.
```

Comment:
you might consider updating the description

Reply:
Changed in [e687855](https://github.com/puzzle/puzzle.opnsense/pull/164/commits/e687855297080c7dd81920d83c7f5c0a2db8fedb)